pkg/network/multus/annotation.go

```diff
@@ -33,6 +33,8 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
+const DefaultNetworkCNIAnnotation = "v1.multus-cni.io/default-network"
```

Comment:
It is a good opportunity to document what this represents and means.

Reply:
Done.